app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -27,8 +27,15 @@ class MS.Views.CalendarView extends Backbone.View
       @stepModal.new(@$el.find('#calendar').data('project-id'), @refresh.bind(@),
         date: date.format('YYYY-MM-DD'))
 
+  # Changes the default event render to load in html rather than title only
   eventRender: (calEvent) -> calEvent.html
 
+  # Load custom content inside the rendered event rather than title only
+  eventAfterRender: (calEvent, initialElement) ->
+    $(initialElement).find('.fc-content').append(calEvent.html)
+    $(initialElement).addClass(calEvent.event_classes)
+    $(initialElement).data('step-id', calEvent.model_id)
```

Comment:
Inside this method would be a good place to remove the initial title element if it is not needed.